tests/test_skvbc_fast_path.py

```diff
@@ -78,13 +78,13 @@ def setUp(self):
         We send a new series of K/V writes and make sure they
         have been processed using the slow commit path.
 
-        Finally we check if a known K/V has been executed.
+        Finally the decorator verifies the KV execution.
         """
         bft_network.start_all_replicas()
-        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
-
+        max_size = 1
         for _ in range(10):
-            await skvbc.write_known_kv()
+            client = bft_network.random_client()
+            await tracker.send_tracked_write(client, max_size)
```

Comment:
To make things more interesting from the linearizability perspective, let's also do a `await tracker.send_tracked_read()` after each `await tracker.send_tracked_write()`. I suggest to do this for every test.

Reply:
I was wondering, would it make sense to generalize `run_concurrent_ops()` from `SkvbcChaosTest`, so that we use it for the fast path tests?

Let's pull  `run_concurrent_ops()` it into SkvbcTracker by parameterizing it
and then using it in the fast path tests (instead of sending just 10 sequential writes).